config/locales/rails_admin.es.yml

```diff
@@ -0,0 +1,148 @@
+es:
```

Comment:
Looks like this file also contains some file encoding corruption goodness.  Definitely needs to be fixed.